magstock/site_sections/magstock.py

```diff
@@ -41,3 +41,11 @@ def any_match(group):
 
     def food_purchases(self, session):
         return {'attendees': session.food_purchasers().order_by(Attendee.full_name).all()}
+
```

Comment:
New action for setting the two things they want to collect at checkin.  This is called at checkin.